classes/tools/parameter/analyzer.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace mageekguy\atoum\tools\parameter;
+
+class analyzer
+{
+    public function getTypeHintString(\reflectionParameter $parameter)
```

Comment:
suggestion: because it's compatible with PHP7.1, I think adding the return type could be interesting.

Reply:
Done.